triage/nw_prj_neuron_track_hours/cli.py

```
@@ -0,0 +1,244 @@
+        "workbook": str(xlsx_path),
+        "reconciliation_json": str(recon_path),
+        "review_queue_csv": str(review_path),
+        "preflight_json": str(preflight_path),
+        "carryover_md": str(carry_path),
+    }
+
+    zip_path = None
+    if zip_output:
+        zip_path = out / "Neuron_Track_Hours_April_May_2026_WEBSAFE.zip"
+        with zipfile.ZipFile(zip_path, "w", zipfile.ZIP_DEFLATED) as zf:
+            for label, fp in outputs.items():
```

Ruff (0.15.15)

[warning] Loop control variable `label` not used within loop body

Rename unused `label` to `_label`

(B007)